cmd/skuba/cluster/init.go

```diff
@@ -54,7 +56,15 @@ func NewInitCmd() *cobra.Command {
 				klog.Fatalf("init failed due to error: %s", err)
 			}
 		},
-		Args: cobra.ExactArgs(1),
+		Args: func(cmd *cobra.Command, args []string) error {
+			if len(args) != 1 {
+				return fmt.Errorf("accepts 1 arg, received %d", len(args))
+			}
+			if len(args[0]) == 0 {
+				return fmt.Errorf("Invalid cluster name: expected non empty cluster name")
+			}
```

Comment:
This code would never be executed, please considering move before `if len(args) != 1`, thanks.

Reply:
It looks okay to me:
```
jean-philippe@laptop-dell /t/skuba-test> skuba cluster init "" --control-plane 10.86.0.5
** This is an UNTAGGED version and NOT intended for production usage. **
Error: Invalid cluster name: expected non empty cluster name
```

Reply:
Ugh, okay, I did not think use `""`, smart :100: